.gitignore

```diff
@@ -27,3 +27,7 @@ storybook-static
 # Vercel Files
 .vercel
 .turbo
+
+# ESlint Cache Files
+.eslintmdcache
+.eslintjscache
```

Comment:
what is the value in separating these? the defaults are hidden away and opaque to users. My only guess is the scale of the site benefits a split like this.

Reply:
I would say having a separate cache for the different commands makes sense (we have two different eslint commands) and also separates content from code.